Examples/VoiceMemos/VoiceMemosTests/VoiceMemosTests.swift

```diff
@@ -281,12 +281,44 @@ class VoiceMemosTests: XCTestCase {
     )
 
     store.assert(
-      .send(.deleteVoiceMemo(IndexSet(integer: 1))),
```

Comment:
I don't know why we were removing this index that doesn't even exist in the state. it didn't crash because apparently `remove(atOffsets:)` doesn't crash when you remove at indices that don't exist.